content/news/028/index.md

```diff
@@ -268,6 +268,21 @@ performance and robustness improvements.
 [RFC9000]: https://www.rfc-editor.org/rfc/rfc9000.html
 [quinn_release]: https://github.com/quinn-rs/quinn/releases/tag/0.8.0
 
+### [hecs]
+
+[hecs] is a fast, lightweight, and unopinionated archetypal ECS library.
+
+[Version 0.7](hecs-changelog) introduces several new features, including two new
```

Comment:
Link syntax is wrong here for a footnote-style link:
```suggestion
[Version 0.7][hecs-changelog] introduces several new features, including two new
```

Reply:
Good catch, I just resolved it on `source` :+1: